engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/details/views/content-host-info.html

```diff
@@ -41,11 +41,11 @@ <h4 translate>Basic Information</h4>
            bst-feature-flag="remote_actions">
         <span class="info-label" translate>Katello Agent</span>
         <span class="info-value" ng-show="host.content_facet_attributes.katello_agent_installed">
-          <span class="fa fa-circle green"></span>
+          <span class="green host-status pficon pficon-ok status-ok'"></span>
           <span translate>Installed</span>
         </span>
         <span class="info-value" ng-hide="host.content_facet_attributes.katello_agent_installed">
-          <span class="fa fa-circle yellow"></span>
+          <span class="yellow host-status pficon pficon-info status-warn"></span>
```

Comment:
If you used the function here instead of hard-coding, this would be more robust against any future changes since you'd only have to change the content-hosts-helper.server.js code.